### PR TITLE
Add more documentation

### PR DESCRIPTION
**Pull request by eirslett**

### CODE REVIEW

**Comment by sveisvei** [comment]
lgtm

File: README.md [hunk 1]
```diff
@@ -10,62 +10,49 @@ new JavaScript features and syntax, so Node.js version 6 or newer is required.
...
 
-## Transports
-
-You can choose between multiple transports; for now,
-scribe and kafka transports are implemented. They can
-be used like this:
+- cujojs/rest (zipkin-instrumentation-cujojs-rest)
+- express (zipkin-instrumentation-express)
+- fetch (zipkin-instrumentation-fetch)
+- memcached (zipkin-instrumentation-memcached)
```
Comment (adriancole) [inline README.md]: heh that's new right? cool!

File: README.md [hunk 2]
```diff
@@ -10,62 +10,49 @@ new JavaScript features and syntax, so Node.js version 6 or newer is required.
...
+Every module has a README.md file that describes how to use it.
 
-```javascript
-const ScribeTracer = require('zipkin-transport-scribe');
-tracer.pushTracer(new ScribeTracer({
-  host: 'localhost',
-  port: 9410
-});
-```
+## Transports
```
Comment (adriancole) [inline README.md]: note we've been using the name "reporter" for the client-side in latest arch docs, though maybe late to change..

Comment (adriancole) [inline README.md]: recorder is a term used in OpenTracing more often for the thing that aggregates spans from pieces, then reports them onto a transport. We chose "reporter" in the arch to handle the piece that's different between kafka and scribe

File: packages/zipkin-instrumentation-fetch/README.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# zipkin-instrumentation-fetch
```
Comment (adriancole) [inline packages/zipkin-instrumentation-fetch/README.md]: should any of these packages be namespaced for node vs not node? I've no opinion, just curious

File: packages/zipkin-instrumentation-fetch/README.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+const rest = require('rest');
+const wrapFetch = require('zipkin-instrumentation-fetch');
+
+const tracer = new Tracer({ctxImpl, recorder}); // configure your tracer properly here
+
+const nameOfRemoteService = 'youtube';
+const zipkinFetch = wrapFetch(fetch, {tracer, serviceName: nameOfRemoteService});
+
+// Your application code here
+zipkinFetch('http://www.youtube.com/').then(res => res.json()).then(data => ...);
```
Comment (adriancole) [inline packages/zipkin-instrumentation-fetch/README.md]: cool!

File: packages/zipkin-transport-kafka/README.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+## Usage:
+
+`npm install zipkin-transport-kafka --save`
+
+```javascript
+const {Tracer, BatchRecorder} = require('zipkin');
+const KafkaLogger = require('zipkin-transport-kafka');
+
+const kafkaRecorder = new BatchRecorder({
```
Comment (adriancole) [inline packages/zipkin-transport-kafka/README.md]: ex here, recorder looks right, but logger is a word we're currently using reporter for

**Comment by adriancole** [comment]
nice work :shipit: my notes were things to ponder

File: packages/zipkin-transport-kafka/README.md (hunk 5 above)
Comment (sveisvei) [inline packages/zipkin-transport-kafka/README.md]: Yeah, lingo needs to be consistant, and might also be nice with an little readme about the lingo :).

**Pull request merged**